.github/pull_request_template.md

```diff
@@ -23,16 +21,21 @@ Addresses #*PR# HERE*
 ## (optional) Implementation notes
 
 ### At a high level, how did you implement this?
-
 <!-- Briefly describe how this works -->
 
 ### Does this introduce any tech-debt items?
-
 <!-- List anything that will need to be addressed later -->
 
+## Testing checklist
+<!-- Complete the checklist before submitting a PR; delete anything that doesn't apply -->
 
-## Reviewer guidance
+- [ ] Contributor has fully tested the PR manually
+- [ ] If there are any front-end changes, before/after screenshots are included
+- [ ] Critical and brittle code paths are covered by unit tests
+- [ ] The change has been added to the `changelog`
+- [ ] A PR has been submitted to point the Kolibri `develop` branch to the most recent version of KDS
```

Comment:
Should we require similar PRs for Studio and KDP?

If that's too onerous, maybe we should actually be looking into auto-generated PRs based on github actions?

Reply:
Hmm. Good point. Should we ask the folks in charge of those projects (or at least the release managers)?

Reply:
posted in Slack here: https://learningequality.slack.com/archives/C01PBEMPZC7/p1625172143079600

Reply:
That makes a lot of sense! I'll update it to reflect that.

Reply:
I'm a little confused on one detail:

If some KDS PR has not been merged yet, what commit would the product update PRs point at? I was expecting that the update PRs would point to the merge commit of the KDS PR.

Reply:
That makes sense but then the question is if it should be in "Reviewer guidance" of the PR template when it's not possible to check it during the review process itself

Reply:
(I wrote my first note assuming that it's something to be done during the review)

Reply:
Ah, that's a good point.

Maybe we should add a post-merge checklist? e.g. something like...

> # Post-merge updates
> 
> After merging, unstabled branches of Kolibri products (Learning Platform, Studio, and Data Portal) should be updated to point at the merge commit resulting from this PR. This process should be led by the submitter of the Design System PR in collaboration with other LE team members working on the other product repos.
>  
> * [ ] Learning Platform update PR is submitted
> * [ ] Studio update PR is submitted
> * [ ] Data Portal update PR is submitted
> * [ ] Learning Platform update PR is merged
> * [ ] Studio update PR is merged
> * [ ] Data Portal update PR is merged


thoughts?

Reply:
Makes much sense to me

Reply:
I appreciate that each product has its own item on the list, could be helpful to keep track of what still needs to be done.

Reply:
FYI, after discussing in Product Tactical, @rtibbles decided that we should not consider the "merged" items mandatory, at least in the shorter-term. We can still include the full list, but the most important things are

1. keeping the changelog up-to-date
2. opening PRs against target products for visibility

Newer updates should supersede older ones in the case where multiple KDS have happened before a product is updated with the latest.

(Longer-term, we can automate opening of PRs with github actions, similar to dependabot)

Reply:
We may want to update the template text to clarify the above